python/pyspark/sql/functions/builtin.py

```diff
@@ -15452,6 +15452,135 @@ def parse_json(
     return _invoke_function("parse_json", _to_java_column(col))
 
 
+@_try_remote_functions
+def is_variant_null(v: "ColumnOrName") -> Column:
+    """
+    Check if a variant value is a variant null. Returns true if and only if the input is a variant
+    null and false otherwise (including in the case of SQL NULL).
+
+    .. versionadded:: 4.0.0
+
```

Comment:
Can we add `Returns` section (https://numpydoc.readthedocs.io/en/latest/format.html#returns)?